lib/stripe.ex

```diff
@@ -3,7 +3,9 @@ defmodule Stripe do
     A HTTP client for Stripe.
     This module contains the Application that you can use to perform
     transactions on stripe API.
-    ### Configuring
+    
+    **Configuring**
+    
```

Comment:
This replaces an `<h3>` in the generated output with a `<strong>` tag.

Ideally, it should be enough of a fix to simply keep the `### Configuring` and and a new line before AND after it.

Did you try that and it did not work?